<?xml version="1.0" encoding="UTF-8"?>
<item>
<layout class="QFormLayout" name="formLayout">
<item row="0" column="0">
<widget class="QLabel" name="labelName">
<property name="text">
<string>Name</string>
</property>
</widget>
</item>
<item row="0" column="1">
<widget class="QLineEdit" name="nameLineEdit"/>
</item>
<item row="1" column="0">
<widget class="QLabel" name="labelQty">
<property name="text">
<string>Quantity</string>
</property>
</widget>
</item>
<item row="1" column="1">
<widget class="QDoubleSpinBox" name="qtySpin">
<property name="minimum">
<double>0</double>
</property>
<property name="maximum">
<double>1000000</double>
</property>
</widget>
</item>
<item row="2" column="0">
<widget class="QLabel" name="labelUnit">
<property name="text">
<string>Unit</string>
</property>
</widget>
</item>
<item row="2" column="1">
<widget class="QLineEdit" name="unitLineEdit"/>
</item>
<item row="3" column="0">
<widget class="QLabel" name="labelTags">
<property name="text">
<string>Tags (comma separated)</string>
</property>
</widget>
</item>
<item row="3" column="1">
<widget class="QLineEdit" name="tagsLineEdit"/>
</item>
</layout>
</item>
<item>
<widget class="QDialogButtonBox" name="buttonBox">
<property name="orientation">
<enum>Qt::Horizontal</enum>
</property>
<property name="standardButtons">
<set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
</property>
</widget>
</item>
</layout>
</widget>
<resources/>
<connections/>
</ui>
